src/test_workflow/integ_test/service.py

```diff
@@ -68,7 +68,7 @@ def get_service_response(self):
     def service_alive(self):
         response = self.get_service_response()
         logging.info(f"{response.status_code}: {response.text}")
-        if response.status_code == 200 and ('"status":"green"' or '"status":"yellow"' in response.text):
+        if response.status_code == 200 and (('"status":"green"' in response.text) or ('"status":"yellow"' in response.text)):
```

Comment:
The service response is a JSON. To future proof this, there should be a class that wraps it, and exposes it’s properties, including an `.alive` method.